calculator/src/App.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+
+import Container from 'react-bootstrap/Container'
+import Col from 'react-bootstrap/Col'
```

Comment:
the 2 import lines from bootstrap can be made into 
import {Col,Row} from 'react-bootstrap';

Reply:
true true!!

Reply:
wait.... actually i do this this way as bootstrap is so large that I heard it was faster to tell the app what to look where than actually saying look this in all bootstrap